recipes/patchelf/all/conanfile.py

```diff
@@ -0,0 +1,54 @@
+from conans import ConanFile, AutoToolsBuildEnvironment, tools
+from conans.errors import ConanInvalidConfiguration
+import os
+
+required_conan_version = ">=1.33.0"
+
+class PatchElfConan(ConanFile):
+    name = "patchelf"
+    description = "A small utility to modify the dynamic linker and RPATH of ELF executables"
+    topics = ("conan", "elf", "linker", "interpreter", "RPATH", "binaries")
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://github.com/NixOS/patchelf"
+    license = "GPL-3.0-or-later"
+    settings = "os", "arch", "compiler", "build_type"
+
+    _autotools = None
+
+    @property
+    def _source_subfolder(self):
+        return "source_subfolder"
+
+    def build_requirements(self):
+        self.build_requires("libtool/2.4.6")
+
+    def configure(self):
+        if not tools.is_apple_os(self.settings.os) and self.settings.os != "Linux":
```

Comment:
`!apple && !linux` is equivalent with `!linux`.
```suggestion
        if self.settings.os not in ("FreeBSD", "Linux"):
```
I added `FreeBSD` to the suggestion.
Don't know whether it works there, but perhaps?

Reply:
[Looks good](https://www.freshports.org/sysutils/patchelf), thanks for the suggestion! 👍